openfisca_us/variables/finance/benefit/inputs.py

```diff
@@ -0,0 +1,24 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class gross_income(Variable):
```

Comment:
This is a very general name so would advise either prepending `snap_` or moving elsewhere if it's general enough